examples/bri-3/src/bri/identity/bpiSubjectAccounts/models/bpiSubjectAccount.ts

```diff
@@ -1,14 +1,12 @@
-import { IBpiSubject } from "../../bpiSubjects/models/bpiSubject.interface";
-import { IBpiSubjectAccount } from "./bpiSubjectAccount.interface";
+import { BpiSubject } from '../../bpiSubjects/models/bpiSubject';
 
-export class BpiSubjectAccount implements IBpiSubjectAccount {
-    id: string; // TODO: Add uuid after #491
-    creatorBpiSubject: IBpiSubject;
-    ownerBpiSubject: IBpiSubject;
-    bpiAccounts: string[]; // TODO: Add as part of #489
+export class BpiSubjectAccount {
+    private id: string; // TODO: Add uuid after #491
+    private creatorBpiSubject: BpiSubject;
+    private ownerBpiSubject: BpiSubject;
+    private bpiAccounts: string[]; // TODO: Add as part of #489
```

Comment:
@Manik-Jain do we want to allow `bpiAccounts` to be an empty array? or rather only require the 1st three elements in the class. The scenario that `bpiAccounts` is known before the `BpiSubjectAccount` is not guaranteed. In fact not likely unless a new user is added to a workstep that uses a `bpiAccounts` element. cc @ognjenkurtic @skosito @Kasshern @Ybittan

Reply:
I agree, let's keep it out of ctor until later.

Reply:
+1

Reply:
+1